requirements.txt

```diff
@@ -6,11 +6,11 @@ kafka-python~=2.0.2
 logzero>=1.7.0
 magic-logger>=1.0.2
 pandas>=1.2.4
-psycopg2-binary~=2.8.6
+psycopg2-binary~=2.9.3
```

Comment:
`psycopg2>3.9.1` is required for `python 3.10`

github issue: https://github.com/psycopg/psycopg2/issues/1371

Reply:
Does this work on both python versions?

Reply:
It does but as discussed in other comments we should also test it during ci!